tests/ui/if_same_then_else.rs

```diff
@@ -232,6 +232,20 @@ fn if_same_then_else() -> Result<&'static str, ()> {
         return Ok(&foo[0..]);
     }
 
+    if true {
+        let foo = "";
+        return Ok(&foo[0..]);
```

Comment:
Shouldn't this `if` body get linted also?

Reply:
it seems that the behavior for the if_arms lint is only targeted to the "then else" parts.
`lint_same_then_else`